### DISCUSSION TITLE
Can't download youtube on chrome, would like a refund

### DISCUSSION THREAD

**Discussion by bizr007** [aclap-dev/video-downloadhelper#1277] [License & Purchase requests]
### What's your issue?

I can't download videos on Chrome, even though it says I can. I would like a refund, I though the license was for multiple browsers, Chrome, Firefox, and Edge.

### The 10 first digits (only 10) of your license number or email address

9f758c6deb

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Sorry, refund is only possible the first month.
